release-notes/schemas/dotnet-patch-releases-index.json

```diff
@@ -0,0 +1,50 @@
+{
+  "description": "A set of product releases with high-level information, like latest version.",
+  "type": [
+    "object",
+    "null"
+  ],
+  "properties": {
+    "channel-version": {
+      "type": "string"
+    },
+    "releases": {
+      "type": "array",
+      "items": {
+        "type": [
+          "object",
+          "null"
+        ],
+        "properties": {
+          "release-version": {
+            "description": "Version of release.",
+            "type": "string"
+          },
+          "release-date": {
+            "description": "Date of release.",
+            "type": "string",
+            "format": "date"
+          },
+          "security": {
+            "description": "Security status of release.",
```

Comment:
This description only makes sense to me if it were an enum - what about something like _"Whether the release contains any CVE fixes"_?